lua/barbar/groups.lua

```diff
@@ -6,29 +6,30 @@ local table_insert = table.insert
 local strcharpart = vim.fn.strcharpart --- @type function
 local strwidth = vim.api.nvim_strwidth --- @type function
 
-local utils = require'barbar.utils'
+--- @class barbar.groups.item
+--- @field hl string the highlight group to use
+--- @field text string the content being rendered
 
 --- @class barbar.groups
-local m = {}
+--- @field [integer] nil|barbar.groups.item
+local groups = {}
 
 --- Sums the width of the groups
---- @param groups barbar.render.group[]
 --- @return integer
-function m.width(groups)
+function groups:width()
```

Comment:
Disagree a lot with this change. Semantically, `:` is used to call a method on a class instance, which is not the case here.

Reply:
I'll drop this part